validators/sn1_validator_wrapper.py

```diff
@@ -36,7 +36,10 @@ async def get_stream_response(self, params: QueryValidatorParams) -> StreamRespo
         # task_name = utils.guess_task_name(params.messages[-1])
 
         # Get the list of uids to query for this step.
-        uids = self.sample_uids(params)
+        if self._query_validators:
+            uids = sample_hotkey_uids(self.validator, [self.validator.wallet.hotkey.ss58_address], k=1)
```

Comment:
is k=1 right to be fixed here?

Reply:
Maybe we can reuse the `get_random_uids` function to reduce repetition